README.md

```diff
@@ -1,10 +1,23 @@
 # ember-headless-form
 
-[Short description of the addon.]
+[![CI](https://github.com/CrowdStrike/ember-headless-form/actions/workflows/ci.yml/badge.svg?branch=main&event=push)](https://github.com/CrowdStrike/ember-headless-form/actions/workflows/ci.yml)
+
+The Ember.js addon that distills the common behavior and accessibility best practices of forms into reusable components, without any opinions on specific markup or styling. Use it to build your forms directly, or to build your opinionated forms component kit on top of it.
```

Comment:
This might confuse things, but I wonder if we should explicitly tell folks to not use the @ember-headless-form prefix?

I don't know which is why I am asking...